docs/guidance/map/related_processes.md

```diff
@@ -10,23 +58,251 @@ A contracting process brings together, under a single identifier, the informatio
+
+### Invitation to participate in the first stage of a framework agreement procedure
+
+* Create a release and set `.tag` to 'tender'
+* For each buyer that will participate in the framework agreement, add an `Organization` object to `parties`, set `.role` to 'buyer' and populate its other fields.
+* If all public entities can participate, then omit them from the parties array and add a note in the publication policy to explain how users should interpret a contracting process with no buyers..
+* If only one buyer will participate in the framework agreement, add an `OrganizationReference` object to `buyer` and set `.id` and `.name` to the values from the buyer's `Organization` object. Otherwise, don't populate `buyer`.
+* In the `tender` section, set:
+    *   `.techniques.hasFrameworkAgreement` to 'True'.
+    *   `.contractPeriod` to the duration of the framework agreement.
+    *   If the framework agreement is closed, set `.tenderPeriod.endDate` to the deadline for responses to the invitation.
+    *   If the framework agreement is open, set `.tenderPeriod.endDate` to the last date that new suppliers can be added.
+
+#### Setting procurement method
+
+The `tender.procurementMethod` field uses the [method codelist](../../schema/codelists.md#method) to describe the competitive conditions of the framework agreement procedure as a whole, not only the first stage.
+
+Use the following criteria to determine the procurement method:
+
+* If the contracting process will establish a framework agreement with a single supplier and if any supplier can submit a request to participate, i.e. there are no qualification criteria, then set `.procurementMethod` to 'open'.
+* If the contracting process limits the suppliers that can submit a request to participate in the framework agreement, then set `.procurementMethod` to 'limited'.
+* Otherwise, set `.procurementMethod` to ‘selective’.
+
+### Addition of a supplier as a party to a framework agreement
+
+* Create a release with the same `ocid` as the 'tender' release and set `.tag` to ‘award’
+* Add an `Award` object to `awards`.
+* For each supplier:
+    *   Add an `Organization` object to `parties`, set `.role` to 'supplier' and populate its other fields.
+    *   Add an `OrganizationReference` object to `awards.suppliers` and set `.id` and `.name` to the values from supplier's `Organization` object.
+* If no further suppliers will be added to the framework agreement, set `tender.status` to 'complete'.
+
+### Award of a procurement contract without second-stage competition
+
+* Create a release with a new `ocid` and set `.tag` to 'award,contract’
+* [Relate the second stage to the first stage](#relate-the-second-stage-to-the-first-stage)
+* [Add a buyer](#add-a-buyer)
+* [Add an award, contract and supplier](#add-an-award-contract-and-supplier)
+* Include the `tender` section with the `tender.id` set to the `award.id`, include the `procuringEntity` if it exists, and set `tender.competitive` to ‘false’  
```

Comment:
What is meant by "Include the `tender` section"? My understanding is that *only* these fields will be set. In my commit, I therefore remove "Include the `tender` section".